openwisp_controller/config/admin.py

```diff
@@ -98,8 +98,10 @@ def get_extra_context(self, pk=None):
                     'title': '{0} (ALT+P)'.format(text),
                 }
             ],
-            'CONFIG_BACKEND_FIELD_SHOWN': app_settings.CONFIG_BACKEND_FIELD_SHOWN,
         }
+        # Do not pass CONFIG_BACKEND_FIELD_SHOWN for Vpn model
+        if self.model.__name__ != 'Vpn':
+            ctx['CONFIG_BACKEND_FIELD_SHOWN'] = app_settings.CONFIG_BACKEND_FIELD_SHOWN
```

Comment:
I changed this to:

```python
# do not pass CONFIG_BACKEND_FIELD_SHOWN in VpnAdmin
# since we don't need to hide the VPN backend there
if not issubclass(self.model, AbstractVpn):
    ctx['CONFIG_BACKEND_FIELD_SHOWN'] = app_settings.CONFIG_BACKEND_FIELD_SHOWN
```

The reason for this change is that ow-controller can be extended and its model can be renamed, so the name of the model may be slightly different than `Vpn`. Using `issubclass` is more reliable because it checks the parent class instead of the name.